test/e2e-go/features/stateproofs/stateproofs_test.go

```diff
@@ -403,6 +402,7 @@ func getStateProofByLastRound(r *require.Assertions, fixture *fixtures.RestClien
 	curRound, err := fixture.LibGoalClient.CurrentRound()
 	r.NoError(err)
 
+	// Note: This uses a deprecated v1 endpoint, and needs to be rewritten to use a v2 equivalent.
```

Comment:
@algochoi Opening a thread as a visual reminder - I think we need to identify a plan before merging the PR.  Though the changes may come in a follow on PR.

Reply:
It _looks_ like https://github.com/algorand/go-algorand/pull/4717 addresses the thread.  Let's leave the thread open until https://github.com/algorand/go-algorand/pull/4717 is merged into `master` + this PR.

@algoidan thanks for identifying an alternate path!

Reply:
I cherry-picked the commit in this commit: [5ccca0d](https://github.com/algorand/go-algorand/pull/4684/commits/5ccca0d6a666fb2c134c5445ee44cdf8a488cfaf)

Reply:
@algochoi Resolving - Thanks for bringing https://github.com/algorand/go-algorand/pull/4717 into the PR.